Source/Core/CesiumTerrainProvider.js

```diff
@@ -484,7 +487,7 @@ define([
      * @exception {DeveloperError} This function must not be called before {@link CesiumTerrainProvider#ready}
      *            returns true.
      */
-    CesiumTerrainProvider.prototype.requestTileGeometry = function(x, y, level, throttleRequests) {
+    CesiumTerrainProvider.prototype.requestTileGeometry = function(x, y, level, request) {
```

Comment:
This needs some backwards compatibility code for when "`request`" is still a boolean.  We'll also deprecate this - over one release is fine.